packages/wrangler/src/pages/dev.ts

```diff
@@ -65,6 +65,11 @@ export function Options(yargs: CommonYargsArgv) {
 				default: 8788,
 				description: "The port to listen on (serve from)",
 			},
+			inspect: {
```

Comment:
Does `wrangler pages` have unit tests that check the help message?
If so, then those would need updating with this new arg.